flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer010.java

```diff
@@ -69,6 +70,12 @@
 
 	private static final long serialVersionUID = 2324564345203409112L;
 
+	/**
+	 * RateLimiter to throttle bytes read from Kafka. The rateLimiter is set via
+	 * {@link #setRateLimiter(FlinkConnectorRateLimiter)}.
+	 */
+	private FlinkConnectorRateLimiter rateLimiter;
```

Comment:
Why repeat this here (already defined in `FlinkKafkaConsumer09`)?

Reply:
Fair point. I removed the field and added a getter to access the `rateLimiter` from the super class. Let me know if that doesn't seem reasonable.